.github/dependabot.yml

```diff
@@ -16,12 +16,4 @@ updates:
   - dependency-name: phpunit/phpunit
     versions:
     - ">= 9.a"
-    - "< 10"
-- package-ecosystem: npm
```

Comment:
Confirming: no reason for NPM on this project, right?

Reply:
Not that I'm aware of. It might be possible to dig through the history to see what it _was_ used for, but I don't think it's used for anything _currently_.